utils/Api.js

```diff
@@ -0,0 +1,110 @@
+class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getAppInfo() {
+    return Promise.all([
+      this.getInitialCards(),
+      this.getUserInfo(),
+      this.editUserInfo,
```

Comment:
`  this.editUserInfo,` is not needed here because it has no effect


```js

  getAppInfo() {
    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
  }


```